runelite-client/src/main/java/net/runelite/client/plugins/fishing/FishingPlugin.java

```diff
@@ -351,6 +364,50 @@ public void onVarbitChanged(VarbitChanged event)
 		}
 	}
 
+	@Subscribe
+	public void onWidgetLoaded(WidgetLoaded event)
+	{
+		if (event.getGroupId() == WidgetID.FISHING_TRAWLER_GROUP_ID)
+		{
+			trawlerStartTime = Instant.now();
+		}
+	}
+
+	/**
+	 * Changes the Fishing Trawler timer widget from minutes to minutes and seconds
+	 */
+	private void updateTrawlerTimer()
+	{
+		int regionID = client.getLocalPlayer().getWorldLocation().getRegionID();
+
+		if (regionID != TRAWLER_SHIP_REGION_NORMAL && regionID != TRAWLER_SHIP_REGION_SINKING || trawlerStartTime == null)
+		{
+			return;
+		}
+
+		Widget trawlerTimerWidget = client.getWidget(WidgetInfo.FISHING_TRAWLER_TIMER);
+
+		long timeLeft = TRAWLER_TIME_LIMIT_IN_SECONDS - Duration.between(trawlerStartTime, Instant.now()).getSeconds();
+		int minutes = (int) timeLeft / 60;
+		int seconds = (int) timeLeft % 60;
+
+		final StringBuilder trawlerText = new StringBuilder();
+		trawlerText.append("Time Left: ");
+
+		if (minutes > 0)
+		{
+			trawlerText.append(' ');
+			trawlerText.append(minutes);
+			trawlerText.append(" Mins");
```

Comment:
I would change this to mm:ss format, this Mins Secs format do not even makes sense

Reply:
Wanted to keep it consistent with the current format but I can change it. Going to test it and get screenshots tonight